tools/jvm/src/main/scala/org/scalajs/core/tools/linker/backend/closure/ClosureLinkerBackend.scala

```diff
@@ -38,7 +38,7 @@ final class ClosureLinkerBackend(
     outputMode: OutputMode,
```

Comment:
But no matter what you chose, you'll get isolated in the end, no? So how we desugar internally before feeding to GCC really is an implementation detail, no?